skr/webhook/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/kyma-project/kyma-watcher/skr/webhook/internal"
+	"github.com/kyma-project/manifest-operator/operator/pkg/util"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"k8s.io/klog/v2"
+)
+
+type ServerParameters struct {
+	port       int    // webhook server port
+	certFile   string // path to TLS certificate for https
+	keyFile    string // path to TLS key matching for certificate
+	tlsEnabled bool   // indicates if TLS is enabled
+}
+
+var parameters ServerParameters //nolint:gochecknoglobals
+
+func main() {
+	logger := ctrl.Log.WithName("skr-webhook")
+	var err error
+
+	// port
+	portEnv := os.Getenv("WEBHOOK_PORT")
+	defaultPort := 8443
+	if portEnv != "" {
+		defaultPort, err = strconv.Atoi(portEnv)
+		if err != nil {
+			logger.Error(err, "Error parsing Webhook server port")
+		}
+	}
+	parameters.port = defaultPort
+
+	// tls
+	tlsEnabledEnv := os.Getenv("TLS_ENABLED")
+	tlsEnabled := false
+	if tlsEnabledEnv != "" {
+		tlsEnabled, err = strconv.ParseBool(tlsEnabledEnv)
+		if err != nil {
+			logger.Error(err, "Error parsing tls flag")
+		}
+	}
+	parameters.tlsEnabled = tlsEnabled
+	parameters.certFile = os.Getenv("TLS_CERT")
+	parameters.keyFile = os.Getenv("TLS_KEY")
+
+	// rest client
+	restConfig, err := util.GetConfig("", "")
```

Comment:
This call should not be based on a manifest operator util package and we should not import it here as a dependency at all imo, maybe just copy over the code instead

Reply:
yes Im aware, that was the plan anyways.
first quick implementation :D

Reply:
https://github.com/kyma-project/kyma-watcher/issues/42